faktiva_cleanurls.php

```diff
@@ -16,6 +16,8 @@
  * @author   Emiliano 'AlberT' Gabrielli <albert@faktiva.com>
  * @license  https://creativecommons.org/licenses/by-sa/4.0/  CC-BY-SA-4.0
  * @source   https://github.com/faktiva/prestashop-clean-urls
+ * @modified for Prestashop 1.7 by Juan F. Molina <juan@juanmolina.es>
+ * @source https://github.com/juferlover/prestashop-clean-urls/tree/Prestashop-1.7-Update
```

Comment:
Hi, thanks for the PR but adding credits to every GH contributor in the docblock is not in my policy ... if you really want explicit credits, in addition to the commit contributors in GH, you can add a "Contributors.md" file instead.

Thanks